packages/controls/src/widget_string.ts

```diff
@@ -292,6 +296,13 @@ class TextView extends DescriptionView {
         this.textbox.setAttribute('placeholder', value);
     }
 
+    update_title() {
+        let title = this.model.get('description_tooltip');
+        if (title.length > 0) {
+            this.textbox.setAttribute('title', title);
+        }
```

Comment:
If the title length is now 0, we should probably unset the textbox title attribute.